lib/ui/contact-list.dart

```diff
@@ -0,0 +1,96 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_contact_list/ui/contact-detail.dart';
```

Comment:
File name should be 'contact_detail_screen.dart':
https://dart.dev/guides/language/effective-dart/style